pkg/export/disk/disk.go

```diff
@@ -116,6 +119,7 @@ func (r *Writer) UpdateConnection(_ context.Context, connectionName string, conf
 	}
 
 	conn.MaxAuditResults = int(maxResults)
+	conn.ClosedConnectionTTL = ttl
```

Comment:
will this cause inconsistent behavior if user updates an existing failed connection? do we want to loop over closed connections?